integration-tests/jest-reporter.js

```diff
@@ -32,7 +32,8 @@ module.exports = class JestReporter {
     const writableStream = createWriteStream(logFileName)
     writableStream.on('error', ({ message }) => console.error(`Error occurred while writing Console logs to ${logFileName}.\n${message}`))
 
-    _.forEach(({ message }) => writableStream.write(`${message}\n`), testResult.console)
+    testResult.console.forEach(({ message }) => writableStream.write(`${message}\n`)
```

Comment:
What is the structure of `testResult.console`? I am curious as to why `_.forEach` missed items

Reply:
> I suspect _.forEach fails to "see" all elements because array is very large

I can't imagine how that'd be and wonder if something else was happening at the same time when the issue occurred. I don't want to block the merge though.